ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/virtual-source-nat.j2

```diff
@@ -1,5 +1,5 @@
 {# eos - virtual source-nat - introduced in 4.21 #}
-{% if virtual_source_nat_vrfs is defined and virtual_source_nat_vrfs is not none %}
+{% if virtual_source_nat_vrfs is arista.avd.defined %}
```

Comment:
Indeed, It is because we start the section with a `!` and it helps to not put complexity in the loop

```
{% if virtual_source_nat_vrfs is arista.avd.defined %}
!
{%     for vrf in virtual_source_nat_vrfs | arista.avd.natural_sort %}
```

Reply:
Got it. Good point